Sources/Instrumentation/Tracing/TracingInstrument.swift

```diff
@@ -17,11 +17,11 @@ import Dispatch
 public protocol TracingInstrument: Instrument {
     var currentSpan: Span? { get }
 
-    func startSpan(named operationName: String, baggage: BaggageContext, at timestamp: DispatchTime) -> Span
+    func startSpan(named operationName: String, context: BaggageContext, at timestamp: DispatchTime) -> Span
```

Comment:
I think we should pass it as context when we can right? only as baggage when naming conflicts